spec/models/room_spec.rb

```diff
@@ -3,5 +3,12 @@
 RSpec.describe Room, type: :model do
   describe "relationships" do
     it { should belong_to :hotel }
+    it { should have_many :guests }
```

Comment:
You can also add `it { should have_many(:guests).through(:stays) }` and `it { should have_many :stays }`

The same additions can be made in the guest_spec